website/content/labs/lab5.md

```diff
@@ -183,9 +183,10 @@ As above, you should use the TDD workflow when implementing these new functions.
 
```

Comment:
Also, not sure if this is solvable, but the quotation marks on line 177 don't copy-paste nicely into c strings (`“` and `”` vs `"`)

Reply:
Yeah, some editors will automatically change double-quotes to the fancy open/close ones, but I've changed the double quotes so they're the regular, copy-pasteable kind.